static/script.js

```diff
@@ -18,20 +17,43 @@
   const AGENCY_KEY = "AGBUR";
   const MIN_ZOOM = 6;
 
-  L.tileLayer(
+  // create the tile layer with correct attribution
+
+  var hotLayer = new L.tileLayer('http://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png', 
```

Comment:
`L.tileLayer` is a regular function, not a constructor, so `new` should be be used here.